alf/algorithms/config.py

```diff
@@ -181,48 +181,45 @@ def __init__(self,
         assert priority_replay_beta >= 0.0, ("importance_weight_beta should "
                                              "be non-negative be")
 
-        parameters = dict(
-            root_dir=root_dir,
-            algorithm_ctor=algorithm_ctor,
-            data_transformer_ctor=data_transformer_ctor,
-            data_transformer=None,  # to be set by Trainer
-            random_seed=random_seed,
-            num_iterations=num_iterations,
-            num_env_steps=num_env_steps,
-            unroll_length=unroll_length,
-            unroll_with_grad=unroll_with_grad,
-            use_rollout_state=use_rollout_state,
-            temporally_independent_train_step=temporally_independent_train_step,
-            num_checkpoints=num_checkpoints,
-            confirm_checkpoint_upon_crash=confirm_checkpoint_upon_crash,
-            load_checkpoint_strict=load_checkpoint_strict,
-            evaluate=evaluate,
-            eval_interval=eval_interval,
-            epsilon_greedy=epsilon_greedy,
-            eval_uncertainty=eval_uncertainty,
-            num_eval_episodes=num_eval_episodes,
-            summary_interval=summary_interval,
-            update_counter_every_mini_batch=update_counter_every_mini_batch,
-            summaries_flush_secs=summaries_flush_secs,
-            summary_max_queue=summary_max_queue,
-            metric_min_buffer_size=metric_min_buffer_size,
-            debug_summaries=debug_summaries,
-            profiling=profiling,
-            code_snapshots=code_snapshots,
-            summarize_grads_and_vars=summarize_grads_and_vars,
-            summarize_action_distributions=summarize_action_distributions,
-            summarize_output=summarize_output,
-            initial_collect_steps=initial_collect_steps,
-            num_updates_per_train_iter=num_updates_per_train_iter,
-            mini_batch_length=mini_batch_length,
-            mini_batch_size=mini_batch_size,
-            whole_replay_buffer_training=whole_replay_buffer_training,
-            clear_replay_buffer=clear_replay_buffer,
-            replay_buffer_length=replay_buffer_length,
-            priority_replay=priority_replay,
-            priority_replay_alpha=priority_replay_alpha,
-            priority_replay_beta=priority_replay_beta,
-            priority_replay_eps=priority_replay_eps,
-            num_envs=num_envs)
-        for k, v in parameters.items():
-            self.__setattr__(k, v)
+        self.root_dir = root_dir
```

Comment:
The reason of making these changes is so that vscode can know the attributes of config for error reporting and hint.